changelogs/fragments/72876-setup-facts-add-redhat-vendor.yml

```diff
@@ -0,0 +1,4 @@
+bugfixes:
+- "Setup virtualization_facts - add RHV and oVirt type (https://github.com/ansible/ansible/pull/72876)."
+- "This change will fully work from the 4.4.6 oVirt engine, for lower versions, there was a change
```

Comment:
The patch was abandoned, so please remove the sentence. I'd just mention that this works only oVirt/RHV 4.4

Reply:
This change will fully work for VMs in clusters at cluster level 4.4 or newer